test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/JSNativeInPackage.scala

```diff
@@ -0,0 +1,70 @@
+package org.scalajs.testsuite.jsinterop
+
+import org.junit.Test
+import org.junit.Assert._
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSName
+
+package object jsNativeInPackage {
```

Comment:
Uh oh. I just realized this package object has the same name as the class below except for the case. This will make Windows go berserk.

You should rename the package object (and, btw, give it an all-lowercase name). I suggest `packageobjectwithnatives`.